include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -159,11 +159,7 @@ struct GemmKernel
 
     CK_TILE_HOST static bool IsSupportedArgument(const GemmKernelArgs& kargs)
     {
-        constexpr bool is_output_c_reg_transposed =
-            EpiloguePipeline::IsOutputTransposed() != GemmPipeline::IsTransposeC();
-        if constexpr(!((GemmPipeline::VectorSizeC % 2 == 0 &&
-                        std::is_same_v<CLayout, tensor_layout::gemm::RowMajor> &&
-                        is_output_c_reg_transposed) ||
+        if constexpr(!(EpiloguePipeline::GetVectorSizeC() % 2 == 0 ||
```

Comment:
Could you please make this condition more readable by not using negations? I assume that what you want to check here is if: 

1. data type **is** either fp16_t or bf16_t (you could reuse here `is_any_of` https://github.com/ROCm/composable_kernel/blob/1ac48399ec2be5d987b963d7733f734010667b1b/include/ck_tile/core/utility/type_traits.hpp#L123

Then if KBatch != 1 **and** output vector size % 2 != 0 -> false